tests/test_core/test_alignment.py

```diff
@@ -2467,6 +2477,18 @@ def test_slice_align_info(self):
 class AlignmentTests(AlignmentBaseTests, TestCase):
     Class = Alignment
 
+    def test_sliced_deepcopy(self):
+        """correctly deep copy aligned objects in an alignment"""
+        path = "data/brca1_5.paml"
+        aln = load_aligned_seqs(path, array_align=False, moltype="dna")
+        aln = self.Class(aln.named_seqs)
+        aln = aln[20:30]
+        for name in ["Mouse", "Human", "HowlerMon", "NineBande", "DogFaced"]:
```

Comment:
the loop here is incorrect. You need to create 2 copied objects, e.g. `sliced = aln.deepcopy(sliced=True)` and `notsliced = ...` . Then test that `len(notsliced.get_seq('seq1')) == len(orig.get_seq('seq1')` and `len(sliced.get_seq('seq1')) != len(orig.get_seq('seq1')`